foundation-extensions/adobe-experience-platform-assurance/README.md

```diff
@@ -187,30 +169,6 @@ _Note_ For `iOS` using `cocoapods`, run:
 ### Register AEPAssurance with Mobile Core
 
 {% tabs %}
-{% tab title="Android" %}
```

Comment:
We need to keep this tab for Registering the extension since we reference it in the React native tab. We can then move this under React Native tab and clean this up for all extensions in a separate PR.

Reply:
good point.